bzt/requests_model.py

```diff
@@ -89,7 +89,12 @@ def __init__(self, config, scenario, engine):
                 raise TaurusConfigError("Items from upload-files must specify parameter name")
 
             path_exc = TaurusConfigError("Items from upload-files must specify path to file")
-            file_dict["path"] = get_full_path(self.engine.find_file(file_dict.get("path", path_exc)))
+            path = str(file_dict.get("path", path_exc))
+            if not path.startswith('$'):    # exclude varables
```

Comment:
Path can contain variable pattern in the middle. Also, `$` is not enough to tell if this is JMeter variable. We have correct variable detecting somewhere in our code.